helix-term/src/config.rs

```diff
@@ -42,22 +43,22 @@ fn parsing_keymaps_config_file() {
         toml::from_str::<Config>(sample_keymaps).unwrap(),
         Config {
             keys: Keymaps(hashmap! {
-                Mode::Insert => hashmap! {
+                Mode::Insert => Keymap::new(hashmap! {
```

Comment:
I am thinking of removing the inner `hashmap!`

Reply:
Yeah that test was broken, fixed it